src/main/scala/bio/alignment/alignment.scala

```diff
@@ -85,10 +88,10 @@ trait SparseAlignment[T] {
   /**
    * Remove columns, i.e. columns that contain less than min_symbols
    */
-  def removeSparseColumns(in: Alignment, min_symbols: Int): (Alignment, List[Int]) = {
+  def removeSparseColumns(in: List[List[T]], min_symbols: Int): (Alignment, List[Int]) = {
```

Comment:
Why change the type from Alignment to List[List[T]]? I think the former is more readable given how the rest of the API for SparseAlignment is designed.

Reply:
It should've been for test porposes. I corrected it.